installing/installing-fips.adoc

```diff
@@ -83,4 +83,6 @@ If you are using Azure File storage, you cannot enable FIPS mode.
 
 To apply `AES CBC` encryption to your etcd data store, follow the xref:../security/encrypting-etcd.adoc#encrypting-etcd[Encrypting etcd data] process after you install your cluster.
 
-If you add {op-system-base} nodes to your cluster, ensure that you enable FIPS mode on the machines before their initial boot. See xref:../machine_management/adding-rhel-compute.adoc#adding-rhel-compute[Adding RHEL compute machines to an {product-title} cluster] and link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/security_guide/chap-federal_standards_and_regulations#sec-Enabling-FIPS-Mode[Enabling FIPS Mode] in the {op-system-base} 7 documentation.
+If you add {op-system-base} nodes to your cluster, ensure that you enable FIPS mode on the machines before their initial boot. See xref:../machine_management/adding-rhel-compute.adoc#adding-rhel-compute[Adding RHEL compute machines to an {product-title} cluster].
```

Comment:
For Rhel nodes scaleup, we only support Rhel7 .9 for OCP 4.3-OCP4.8. Suggest don't update this part Ahere.
Scale up Rhel8 will only be supported since OCP4.9.

Reply:
These docs changes are for 4.9, right? I assume we make different changes per version of the docs and this wouldn't have to be backported to earlier versions.

AFAIK we continue to support both RHEL 7 and RHEL 8 workers in 4.9, and I'm assuming both can be enabled for FIPS

Reply:
@sosiouxme You are correct, for OCP4.9, we can scaleup both RHEL 7 and RHEL 8.
But for previous releases, only support scaleup RHEL7.
Per https://github.com/openshift/openshift-docs/pull/35858#issue-721074174, the doc should be 4.6+.

Reply:
@chinmayi-chandrasekar I don't think the FIPS enable doc should be removed. Customer still need to refer to the doc for fips enable for RHEL 7 node before OCP packages installed. Otherwise the RHEL 7 scalup will fail.